classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def intitialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student.push(student)
```

Comment:
- [ ] **[REQUIRED]** There is a typo error on line `11`.  you have `@student.push(student)` but your instance variable is `@students`. Let's correct the typo to `@students.push(student)`

- [ ] **[OPTIONAL]** We can use `<<` to push `student` to the `@students` instance variable instead of `push`. It's a shortcut for `push` in Ruby 😄. So it would look like this: `@students << student`

Reply:
Thank you for the recommendation! I have made the changes requested!